python_examples/rebound.py

```diff
@@ -127,4 +149,158 @@ def step():
 def integrate(tmax):
     libias15.integrate(c_double(tmax))
 
+TWOPI = 2.*numpy.pi
+def mod2pi(f):
+    while f<0.:
+        f += TWOPI
+    while f>TWOPI:
+        f -= TWOPI
+    return f
+
+def get_E(e,M):
+    E = M if e<0.8 else numpy.pi
+    
+    F = E - e*numpy.sin(M) - M
+    for i in range(100):
+        E = E - F/(1.0-e*numpy.cos(E))
+        F = E - e*numpy.sin(E) - M
+        if numpy.fabs(F)<1e-16:
+            break
+    E = mod2pi(E)
+    return E
+
+def init_planet(star,       # central object (rebound.Particle object)
+                a,          # semimajor axis
+                e=None,     # eccentricity
+                omega=None, # argument of pericenter
+                f=None,     # true anomaly
+                inc=None,   # inclination
+                Omega=None, # longitude of ascending node
+                mass=None,  # planet mass
+                M=None):    # mean anomaly
+    '''Returns a particle structure to add into rebound for a given set of
+        orbital elements. 'star' and 'a' are required, others will default to zero
+        (with a warning). Only f OR M should be passed to specify the location of
+        the particle on its orbit.'''
+    
+    # first check for valid inputs and give warnings on defaults
+    if e == None: e = 0.; print('e not passed, setting to 0... (circular)')
```

Comment:
I don't think I would print out these warning messages. It seems pretty obvious that if you don't pass `e`, it will be 0 (rather than 0.2348764 or 1). Anyone still not sure what it does can look up your documentation of the function.